docs/pt/docs/advanced/websockets.md

```diff
@@ -0,0 +1,257 @@
+# WebSockets
+
+Você pode usar  <a href="https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API" class="external-link" target="_blank">WebSockets</a> com **FastAPI**.
+
+## Instalando `WebSockets`
+
+Garanta que você criou um [ambiente virtual](../virtual-environments.md){.internal-link target=_blank}, o ativou e instalou o `websockets`:
+
+<div class="termy">
+
+```console
+$ pip install websockets
+
+---> 100%
+```
+
+</div>
+
+## Cliente WebSockets
+
+### Em produção
+
+Em seu sistema de produção, você provavelmente tem um frontend criado com um framework moderno como React, Vue.js ou Angular.
+
+E para comunicar usando WebSockets com seu backend, você provavelmente usaria as utilidades do seu frontend.
+
+Ou você pode ter um aplicativo móvel nativo que se comunica diretamente com seu backend WebSocket, em código nativo.
+
+Ou você pode ter qualquer outra forma de comunicar com o endpoint WebSocket.
+
+---
+
+Mas para este exemplo, usaremos um documento HTML muito simples com algum JavaScript, tudo dentro de uma string longa.
+
+Esse, é claro, não é o ideal e você não o usaria para produção.
+
+Na produção, você teria uma das opções acima.
+
+Mas é a maneira mais simples de focar no lado do servidor de WebSockets e ter um exemplo funcional:
+
+```Python hl_lines="2  6-38  41-43"
+{!../../docs_src/websockets/tutorial001.py!}
+```
+
+## Criando um `websocket`
+
+Em sua aplicação **FastAPI**, crie um `websocket`:
+
+```Python hl_lines="1  46-47"
+{!../../docs_src/websockets/tutorial001.py!}
+```
+
+/// note | "Detalhes Técnicos"
+
+Você também poderia usar `from starlette.websockets import WebSocket`.
+
+A **FastAPI** fornece o mesmo `WebSocket` diretamente apenas como uma conveniência para você, o desenvolvedor. Mas ele vem diretamente do Starlette.
+
+///
+
+## Aguardar por mensagens e enviar mensagens
+
+Em seu rota WebSocket você pode `await` por mensagens e enviar mensagens.
```

Comment:
change to:

Em **sua** rota...

And I think it's a little strange: você pode `await` por mensagens

In english it makes sense, but when translating, it loses the sense...

I think you can translate using something like:

Em sua rota WebSocket você pode esperar (`await`) por mensagens e enviar mensagens.